packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -530,6 +530,12 @@ export default class PrepaidCard {
     };
   }
 
+  async getPrepaidCardSafesFromTxHash(txHash: string): Promise<PrepaidCardSafe[]> {
```

Comment:
As of cardpay-sdk 0.23.0, the PrepaidCard sdk's `create` method has an overlaps that can accept a single txnHash argument that does what this method does. When you have a chance, please rebase and update your code to use that approach.

Reply:
Nice! Will do.